java/src/jmri/InstanceManager.java

```diff
@@ -238,6 +255,42 @@
         return object;
     }
 
+    /**
+     * Retrieve the last object of specific type that was registered with
+     * {@link #store(java.lang.Object, java.lang.Class) }.
+     *
+     * Intended for use with i.e. scripts where access to the class type is inconvenient.
+     * In Java code where typing is enforced, use {@link #getDefault(Class)}.
+     *
+     * <p>
+     * Unless specifically set, the default is the last object stored, see the
+     * {@link #setDefault(java.lang.Class, java.lang.Object) } method.
+     * <p>
+     * In some cases, InstanceManager can create the object the first time it's
+     * requested. For more on that, see the class comment.
+     * <p>
+     * In most cases, system configuration assures the existence of a default
+     * object, so this method will log and throw an exception if one doesn't
+     * exist. Use {@link #getNullableDefault(java.lang.Class)} or
+     * {@link #getOptionalDefault(java.lang.Class)} if the default is not
+     * guaranteed to exist.
+     *
+     * @param className Fully qualified class name
+     * @return The default object for type
+     * @throws NullPointerException if no default object for type exists
+     * @see #getNullableDefault(java.lang.Class)
+     * @see #getOptionalDefault(java.lang.Class)
+     */
+    @Nonnull
+    public static Object getDefault(@Nonnull String className) {
+        log.trace("getDefault of type {}", className);
+        Object object = InstanceManager.getNullableDefault(className);
```

Comment:
I think it would be good to add `@throws IllegalArgumentException` to the Javadoc for this method.
If the caller catches the NPE, it will still have a non catched IAE if the class name is wrong.